ksql-execution/src/main/java/io/confluent/ksql/execution/context/QueryContext.java

```diff
@@ -43,12 +48,6 @@ private QueryContext(List<String> context) {
     }
   }
 
-  @SuppressWarnings("unused")// Invoked via reflection by Jackson
-  @JsonCreator
```

Comment:
For some reason Jackson complains it can't deserialize into ImmutableList when using the JsonCreator (even though it ultimately just calls this constructor). So I switched this class to use a custom deserializer.

Reply:
It's because there is a getter called `getContext`, so its checking out the `private final ImmutableList<String> context;` field.  Mark `getContext` as `@JsonIgnore` and you'll be golden.